build.gradle

```diff
@@ -210,6 +212,9 @@ dependencies {
   functionalTestImplementation sourceSets.main.runtimeClasspath
 
   smokeTestImplementation sourceSets.main.runtimeClasspath
+
+  runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.16'
```

Comment:
💪 
```suggestion
  runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.19'
```

Reply:
👍

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/3b1010d4a605094d298f60eaccff89fdb3ad73b4